grepper_python/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""
+Python Grepper API
+~~~~~~~~~~~~~~~~~~~
+An API wrapper for the Grepper API.
+"""
+
+__title__ = "grepper-python"
+__author__ = "CodeGrepper"
+__license__ = "MIT"
+__copyright__ = "Copyright 2010-2023 Grepper, Inc."
+__version__ = "0.0.1a"
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+import logging
+from typing import NamedTuple, Literal
+
+from .answer import GrepperAnswer
+from .main import Grepper
```

Comment:
```suggestion
from grepper_python.answer import GrepperAnswer
from grepper_python.main import Grepper
```

Also you should place all things available to import inside a `__all__` statement (including the exceptions!)